source/databases-collections.txt

```diff
@@ -0,0 +1,281 @@
+
+      test_collection
+      example_collection
+
+.. TODO: Uncomment when cursors page is created
+.. For more information about iterating over a cursor, see :ref:`c-cursors`. 
+
+Delete a Collection
+-------------------
+
+You can delete a collection from the database by using the ``mongoc_collection_drop()``
+function.
+
+The following example deletes the ``"test_collection"`` collection:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.c
+   :start-after: start-delete-collection
+   :end-before: end-delete-collection
+   :language: c
+   :copyable:
+   :dedent:
+
+.. warning:: Dropping a Collection Deletes All Data in the Collection
+
+   Dropping a collection from your database permanently deletes all
+   documents and all indexes within that collection.
+
+   Drop a collection only if the data in it is no longer needed.
+
+.. _c-config-read-write:
+
+Configure Read and Write Operations
+-----------------------------------
+
+You can control how the driver routes read operations by setting a **read preference**.
+You can also control options for how the driver waits for acknowledgment of
+read and write operations on a replica set by setting a **read concern** and a
+**write concern**.
+
+By default, databases inherit these settings from the ``mongoc_client_t`` instance,
+and collections inherit them from the database. However, you can change these
+settings on your database by using one of the following functions:
+
+- ``mongoc_database_set_read_prefs()``
+- ``mongoc_database_set_read_concern()``
+- ``mongoc_database_set_write_concern()``
+
+The following example shows how to change the read preference of a database by using the
+``mongoc_database_set_read_prefs()`` function:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.c
+   :start-after: start-database-read-prefs
+   :end-before: end-database-read-prefs
+   :language: c
+   :copyable:
+   :dedent:
+
+You can change the read and write settings on your collections by using one of the following functions:
+
+- ``mongoc_collection_set_read_prefs()``
+- ``mongoc_collection_set_read_concern()``
+- ``mongoc_collection_set_write_concern()``
+
+The following example shows how to change the read preference of a collection by using the
+``mongoc_collection_set_read_prefs()`` function:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.c
+   :start-after: start-collection-read-prefs
+   :end-before: end-collection-read-prefs
+   :language: c
+   :copyable:
+   :dedent:
+
+.. tip::
+
+   To see the types of available read preferences, see the
+   `API documentation <{+api-libmongoc+}/mongoc_read_prefs_t.html#read-modes>`__. 
+
+To learn more about the read and write settings, see the following guides in the
+{+mdb-server+} manual:
+
+- :manual:`Read Preference </core/read-preference/>`
+- :manual:`Read Concern </reference/read-concern/>`
+- :manual:`Write Concern </reference/write-concern/>`
```

Comment:
S: since these lines also provide additional resources, you could move to the same tip admonition as above (with a newline between the API doc link and the manual links)